client/src/components/core/App/script.js

```diff
@@ -154,17 +163,23 @@ export default {
 
     initialDataLoaded(num_timesteps, itemId) {
       this.numLoadedGalleries += 1;
-      if (this.dataLoaded) {
-        return;
-      }
+      if (this.dataLoaded)
+          return
```

Comment:
Why this change? Usually its better to keep the braces as it avoids issues with added another line to the if block without adding the braces ....

Reply:
No good reason, just bad habit. I've fixed this now.